src/vs/workbench/contrib/chat/browser/media/chat.css

```diff
@@ -925,6 +925,51 @@ have to be updated for changes to the rules above, or to support more deeply nes
 	border-color: var(--vscode-notificationsWarningIcon-foreground);
 }
 
+/**
+ * Styles for the `prompt instructions` attachment widget.
+ */
+.chat-attached-context .chat-prompt-instructions-attachments {
+	display: flex;
+	flex-direction: column;
+	align-items: flex-start;
+	gap: 2px;
+}
+.chat-attached-context .chat-prompt-instructions-attachment {
+	display: flex;
+	gap: 4px;
+	border-color: currentColor;
+}
+.chat-attached-context .chat-prompt-instructions-attachment .codicon {
+	color: inherit;
+	text-decoration: none;
+}
+.chat-attached-context .chat-prompt-instructions-attachment .chat-implicit-hint {
+	opacity: 0.7;
+	font-size: .9em;
+}
+.chat-attached-context .chat-prompt-instructions-attachment.warning {
+	color: var(--vscode-notificationsWarningIcon-foreground);
+}
+.chat-attached-context .chat-prompt-instructions-attachment.error {
+	color: var(--vscode-notificationsErrorIcon-foreground);
+}
+.chat-attached-context .chat-prompt-instructions-attachment.disabled {
+	border-style: dashed;
+	opacity: 0.75;
```

Comment:
Why does this get a lower opacity when disabled when the implicit widget doesn't? We might have avoided doing that for color contrast accessibility reasons.

Reply:
Thought it is a good idea to have a dimmed reference if it is disabled since it is a common UI language users are used to and to reduce visual noise 🤔 
The `0.75` value is larger than some of the elements in the "normal" mode (e.g., the `.chat-implicit-hint` above) so the element remains prominent enough to not cause significant issues 🔍